virtcontainers/device/config/config.go

```diff
@@ -278,3 +275,36 @@ func GetHostPath(devInfo DeviceInfo) (string, error) {
 
 	return filepath.Join("/dev", devName.String()), nil
 }
+
+// getBackingFile is used to fetch the backing file for the device.
+func getBackingFile(devInfo DeviceInfo) (string, error) {
+	backingFilePath := filepath.Join(getSysDevPath(devInfo), "loop", "backing_file")
+	if _, err := os.Stat(backingFilePath); err != nil {
```

Comment:
You could remove the `Stat` by checking the error from `ReadFile()` using `os.IsNotExist(err)`.